docs: add readme (#30)
tact-lang/defi-cookbook: README.md

## Overview

> This project includes a set of examples for common use cases in Tact. This includes:

Reviewer: ```suggestion
This project includes a set of examples for common use cases in Tact and consists of:
```

- Smart contracts written in the Tact language.
- A TypeScript + Jest testing environment with `@ton/sandbox`.

> - Examples of off-chain and on-chain integrations with Jettons and other TEPs in the future.

Reviewer: ```suggestion
- Examples of off-chain and on-chain integrations with Jettons. Other TEPs are planned too.
```
